content/chainguard/chainguard-images/working-with-images/compiled-programs.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Choosing an Image for your Compiled Programs"
+linktitle: "Images for Compiled Programs"
+aliases: 
+type: "article"
+description: "An overview comparing various Chainguard Images for compiled programs"
+date: 2024-07-12T17:55:01+00:00
+lastmod: 2024-07-12T18:23:31+00:00
+draft: false
+tags: ["Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 200
+toc: true
+---
+
+When selecting the right base image for your application, there are a variety of factors to take into consideration. For starters, it is critical that your application has all of the dependencies it needs to run. The ideal base image will contain the essential packages you need, while leaving out the ones you don’t. However, in practice, you will need to build upon your images so they meet your specific needs, making it all the more important that you have a strong foundation.
+
+In this guide, we will explore a variety of Chainguard Images which are suitable for different compiled applications. We will take a look at their availability, packages, and use-case differences so you can move closer to settling on the best base image for your specific needs.
+
+## Available Images
+
+### static
+The Chainguard `static` base image is a Wolfi-based image available in one variant with the `:latest` tag. The `static` image is extremely minimal and is not intended to be run directly. It is used to host stand-alone, static binaries, like those produced by compilers such as `gcc`, `go`, and `rust`. It does not contain any programs you can run out-of-the-box. You must add your own static binaries to the image, for example using a Dockerfile multi-stage build.
```

Comment:
Let's link to image overviews in our catalog when we introduce each image (you can keep the monospace).

I kind of feel that in our markdown, we should add a blank line after headings for readability. But I doubt it's in our style guide. I'll bring it up with @SharpRake . Of course it makes no difference in the rendered output.

Reply:
We just removed all of the image overviews 😓 I could link out to the registry? images.chainguard.dev

+1 to adding the whitespace as a style convention, I think it looks better aswell

Reply:
Yes, I think link out to the registry. I don't think there will be issues with that, but let me know if I'm missing anything.